.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="JavaScriptLibraryMappings">
-    <file url="file://$PROJECT_DIR$" libraries="{Node.js Core}" />
```

Comment:
My editor automatically deleted this, so I don't believe this changes much. If I understand correctly this is for IntelliJ's old Node indexing behavior (before IntelliJ started using `@types/node` for the shims)